.github/ISSUE_TEMPLATE/icon_update.yml

```
@@ -8,11 +8,11 @@ body:
     attributes:
       value: |
         Before opening a new issue, make sure it isn't covered by an existing issue.
-        Please search for [issues with the `icon outdated` label][icon-outdated-issues]
+        Please search for [issues with the `update-icon/data` label][update-icon/data]
         (including closed issues) before you continue. If you find one for the brand
         you're reporting then leave a comment on it or add a reaction.
 
-        [icon-outdated-issues]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22icon+outdated%22
+        [update-icon/data]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22update+icon%2Fdata%22
```

This URL works but seems more correct to me using ` ` (space, %20):

```suggestion
        [update-icon/data]: https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+label%3A%22update%20icon%2Fdata%22
```

---

The `+` is more correct. Even GitHub is using the `+` for spaces. You can go to our issues page and select a label for the filter. The URL will be:

```
https://github.com/simple-icons/simple-icons/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3Aassessing
```

---

Looks like a crazy decission of GH to me but not a big deal.